### PR TITLE
Adopt Epimorphics standard build pattern

### PR DESCRIPTION
**Pull request by ijdickinson**
epimorphics/hmlr-linked-data#54

This commit updates lr-landing to use Epimorphics standard build
pattern. In particular:

- update `Makefile` to own setting Alpine and Ruby versions
- update `Dockerfile` to expect Alpine and Ruby versions to be passed
  in, and to copy files explicitly rather than use `COPY . .`
- general dependencies update
- fix Rubocop warnings from dependencies update
- update to `sentry-rails` rather than `sentry-raven`
- use the shared GitHub action definitions

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,50 +23,46 @@ IMAGE?=${NAME}/${STAGE}
...
 GITHUB_TOKEN=.github-token
-NPMRC=.npmrc
 BUNDLE_CFG=${HOME}/.bundle/config
-YARN_LOCK=yarn.lock
 
 all: image
 
 ${BUNDLE_CFG}: ${GITHUB_TOKEN}
-	@./bin/bundle config set rubygems.pkg.github.com ${GRP_OWNER}:`cat ${GITHUB_TOKEN}`
+	@./bin/bundle config set --local rubygems.pkg.github.com ${GRP_OWNER}:`cat ${GITHUB_TOKEN}`
```
Comment (jonrandahl) [inline Makefile:31]: @ijdickinson is the `--local` flag here ok or residual from your local testing?

Comment (ijdickinson) [reply Makefile:31]: It's correct. It will write to `./bundle` in the WORKDIR, rather than to the user's home directory

**Review by jonrandahl — APPROVED** [review]
LGTM!

**Review by der — APPROVED** [review]
Looks fine to me.

**Pull request merged**